contributing/community/releases.rst

```diff
@@ -7,12 +7,12 @@ release and maintain its different versions.
 Symfony releases follow the `semantic versioning`_ strategy and they are
 published through a *time-based model*:
 
-* A new **Symfony patch version** (e.g. 2.8.15, 4.1.7) comes out roughly every
+* A new **Symfony patch version** (e.g. 4.4.9, 5.0.9, 5.1.0) comes out roughly every
```

Comment:
While 5.1.0 strictly is a patch version, I wouldn't list it here (it's a minor release)

Reply:
should we wait 5.1.1 to be released? as this PR target 5.1, i wanted to put a 5.1 patch version

Reply:
Let's use `5.1.1` ... it doesn't exist yet, but it will exist very soon, so it's OK.

Reply:
ok